docs/user-guide/expressions/lists.md

```diff
@@ -1,47 +1,102 @@
-# Lists
+# Lists and Arrays
 
-An expression context we haven't discussed yet is the `List` context. This means simply we can apply any expression on the elements of a `List`.
+`Polars` has first-class support for `List` columns: that is, columns where each row is a list of homogenous elements, of varying lengths. `Polars` also has an `Array` datatype, which is analogous to `numpy`'s `ndarray` objects, where the length is identical across rows.
 
-# Row wise computations
+Note: this is different from Python's `List` object, where the elements can be of any time. Polars can store these within columns, but as a generic `Object` datatype that doesn't have the special list manipulation features that we're about to discuss.
 
-This context is ideal for computing things in row orientation.
+# Powerful *List* manipulation
```

Comment:
Please only use one H1 (# in markdown) for the title and use H2.... (##, ###, ...) for subtitles